package.json

```diff
@@ -51,7 +51,7 @@
   "bugs": "https://github.com/aurora-is-near/aurora-relayer/issues",
   "repository": "https://github.com/aurora-is-near/aurora-relayer.git",
   "dependencies": {
-    "@aurora-is-near/engine": "aurora-is-near/aurora.js#6d37aeb",
+    "@aurora-is-near/engine": "aurora-is-near/aurora.js#0b47447",
```

Comment:
I think it should be https://github.com/aurora-is-near/aurora.js/commit/40a96c1680b29228c6c6b87416470f0617098aac right ?

Reply:
Yes